pyface/qt/__init__.py

```diff
@@ -56,7 +60,7 @@ def prepare_pyqt4():
 for api_name, module in QtAPIs:
     if module in sys.modules:
         qt_api = api_name
-        if qt_api == 'pyqt':
+        if qt_api == 'pyqt' and sys.version_info[0] <= 2:
```

Comment:
In that case, there's the `if/else` clause in `prepare_pyqt4` which handles the exception differently for python 2 and 3 can also be removed.

Or do we want to keep it in case someone explicitly calls `prepare_pyqt4`?